virtualization_15.m

```diff
@@ -0,0 +1,21 @@
+//
+//  virtualization_15.m
+//
+#import "virtualization_15.h"
+
+/*!
+ @abstract Set nestedVirtualizationEnabled. The default is false.
+ */
+void setNestedVirtualizationEnabled(void *config, bool nestedVirtualizationEnabled)
+{
+#ifdef INCLUDE_TARGET_OSX_15
+    if (@available(macOS 15, *)) {
+        VZGenericPlatformConfiguration *platformConfig = (VZGenericPlatformConfiguration *)config;
+        if (VZGenericPlatformConfiguration.isNestedVirtualizationSupported) {
```

Comment:
I would expose `IsNestedVirtualizationSupported` in go, and let users of Code-Hex/vz do the test themselves if they want to.
Without `IsNestedVirtualizationSupported`, a user of Code-Hex/vz has no way to print a nice error message when trying to start a nested VM in an environment where nested virt is not supported.

Reply:
I have update the code

Reply:
I think the Go wrapper also needs to check MacOS version, otherwise there's a case where the called function stub doesn't exist at all?